assets/prototype/infrastructure/services/contextProviders/DateTimeProvider.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External imports
+ */
+import { createContext, useState } from '@wordpress/element';
+
+export const DateTimeContext = createContext({});
+
+const DateTimeProvider = ({ children, id, relatedTickets }) => {
```

Comment:
I don't think we should add related items to this context. Related items should go to and come from he relations manager to avoid duplication.

Reply:
Removed that here: https://github.com/eventespresso/event-espresso-core/pull/1933/commits/c59201ef76b162f2b9f810dd6fcb334c424ef220